Filters:
--- Current
+++ TO BE APPLIED
@@ -1 +1,5 @@
+* Criteria:
+    query: list:maillist@google.com
+  Actions:
+    apply label: maillist
 

Labels:
--- Current
+++ TO BE APPLIED
@@ -0,0 +1 @@
+maillist


cmpDiff:
  apply.GmailConfig{
- 	Labels:  s"",
+ 	Labels:  s"maillist",
- 	Filters: filter.Filters{},
+ 	Filters: filter.Filters{
+ 		(
+ 			s"""
+ 			* Criteria:
+ 			    query: list:maillist@google.com
+ 			  Actions:
+ 			    apply label: maillist
+ 			s"""
+ 		),
+ 	},
  }
